docs/admin/kube-proxy.md

```diff
@@ -11,8 +11,8 @@ notitle: true
 
 The Kubernetes network proxy runs on each node. This
 reflects services as defined in the Kubernetes API on each node and can do simple
-TCP, UDP stream forwarding or round robin TCP, UDP forwarding across a set of backends.
```

Comment:
It's good to keep the white space here.

Reply:
All of this content is from generated source - the PR to add in whitespace here will really need to be in kubernetes/kubernetes, but I'm not entirely sure where. I'll research this further and see where changes need to be made.

Reply:
Thanks @heckj !

Reply:
FYI: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L339-L340

Reply:
@xiangpengzhao https://github.com/kubernetes/kubernetes/pull/53671 has been opened to solve upstream issue here